sdk/identity/Azure.Identity.BrokeredAuthentication/src/InteractiveBrowserCredentialBrokerOptions.cs

```diff
@@ -13,21 +13,29 @@ namespace Azure.Identity.BrokeredAuthentication
     public class InteractiveBrowserCredentialBrokerOptions : InteractiveBrowserCredentialOptions, IMsalPublicClientInitializerOptions
     {
         private IntPtr _parentWindowHandle;
+        private bool _msaPassthrough;
 
         /// <summary>
         /// Creates a new instance of <see cref="InteractiveBrowserCredentialBrokerOptions"/> to configure a <see cref="InteractiveBrowserCredential"/>.
         /// </summary>
         /// <param name="parentWindowHandle">Handle of the parent window the system authentication broker should be docked to.</param>
-        public InteractiveBrowserCredentialBrokerOptions(IntPtr parentWindowHandle) : base()
+        /// <param name="msaPassthrough">A legacy option available only to old Microsoft applications. Should be avoided where possible. Support is experimental.</param>
+        public InteractiveBrowserCredentialBrokerOptions(IntPtr parentWindowHandle, bool msaPassthrough = false) : base()
```

Comment:
New default param is a subtle breaking change (a binary breaking change). Prefer to use a new ctor.

Reply:
Thanks for reviewing and good to know that :) `Azure.Identity.BrokeredAuthentication` is a beta so I thought we needn't worry too much about breaking change. Anyway, I'll let @schaabs 's team decide the final look of the API.